Directory.Packages.props

```diff
@@ -40,6 +40,7 @@
     <PackageVersion Include="System.Memory" Version="4.5.5" />
     <PackageVersion Include="System.Reactive" Version="5.0.0" />
     <PackageVersion Include="System.Runtime.Loader" Version="4.3.0" />
+    <PackageVersion Include="System.Text.Json" Version="6.0.7" />
```

Comment:
Why can't we use [`7.0.2`](https://www.nuget.org/packages/System.Text.Json/7.0.2)?

We use `7.0.0` of `Microsoft.Extensions.*` https://github.com/microsoft/component-detection/pull/551/files#diff-5baf5f9e448ad54ab25a091adee0da05d4d228481c9200518fcb1b53a65d4156R14-R17

Reply:
```
  Microsoft.Common.CurrentVersion.targets(2302, 5): [MSB3277] Found conflicts between different versions of "System.Text.Json" that could not be resolved.
There was a conflict between "System.Text.Json, Version=6.0.0.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51" and "System.Text.Json, Version=7.0.0.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51".
    "System.Text.Json, Version=6.0.0.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51" was chosen because it was primary and "System.Text.Json, Version=7.0.0.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51" was not.
    References which depend on "System.Text.Json, Version=6.0.0.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51" [C:\Program Files\dotnet\packs\Microsoft.NETCore.App.Ref\6.0.16\ref\net6.0\System.Text.Json.dll].
        C:\Program Files\dotnet\packs\Microsoft.NETCore.App.Ref\6.0.16\ref\net6.0\System.Text.Json.dll
          Project file item includes which caused reference "C:\Program Files\dotnet\packs\Microsoft.NETCore.App.Ref\6.0.16\ref\net6.0\System.Text.Json.dll".
            C:\Program Files\dotnet\packs\Microsoft.NETCore.App.Ref\6.0.16\ref/net6.0/System.Text.Json.dll
    References which depend on "System.Text.Json, Version=7.0.0.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51" [].
        C:\Users\jamagee\RiderProjects\component-detection\src\Microsoft.ComponentDetection.Contracts\bin\Debug\netstandard2.0\Microsoft.ComponentDetection.Contracts.dll
          Project file item includes which caused reference "C:\Users\jamagee\RiderProjects\component-detection\src\Microsoft.ComponentDetection.Contracts\bin\Debug\netstandard2.0\Microsoft.ComponentDetection.Contracts.dll".
            C:\Users\jamagee\RiderProjects\component-detection\src\Microsoft.ComponentDetection.Contracts\bin\Debug\netstandard2.0\Microsoft.ComponentDetection.Contracts.dll
```